osu.Game.Rulesets.Osu/Mods/OsuModHidden.cs

```diff
@@ -88,38 +86,38 @@ private void applyState(DrawableHitObject drawableObject, bool increaseVisibilit
                     else
                     {
                         // we don't want to see the approach circle
-                        using (circle.BeginAbsoluteSequence(hitObject.StartTime - hitObject.TimePreempt, true))
+                        using (circle.BeginAbsoluteSequence(hitObject.StartTime - hitObject.TimePreempt))
                             circle.ApproachCircle.Hide();
                     }
 
-                    using (drawableObject.BeginAbsoluteSequence(fadeOut.startTime, true))
-                        fadeTarget.FadeOut(fadeOut.duration);
+                    using (drawableObject.BeginAbsoluteSequence(fadeStartTime))
+                        fadeTarget.FadeOut(fadeDuration);
                     break;
 
                 case DrawableSlider slider:
-                    using (slider.BeginAbsoluteSequence(fadeOut.startTime, true))
-                        slider.Body.FadeOut(fadeOut.duration, Easing.Out);
+                    using (slider.BeginAbsoluteSequence(fadeStartTime))
+                        slider.Body.FadeOut(fadeDuration, Easing.Out);
 
                     break;
 
                 case DrawableSliderTick sliderTick:
-                    using (sliderTick.BeginAbsoluteSequence(fadeOut.startTime, true))
-                        sliderTick.FadeOut(fadeOut.duration);
+                    using (sliderTick.BeginAbsoluteSequence(fadeStartTime))
+                        sliderTick.FadeOut(fadeDuration);
 
                     break;
 
                 case DrawableSpinner spinner:
                     // hide elements we don't care about.
                     // todo: hide background
 
-                    using (spinner.BeginAbsoluteSequence(fadeOut.startTime, true))
-                        spinner.FadeOut(fadeOut.duration);
+                    using (spinner.BeginAbsoluteSequence(fadeStartTime))
+                        spinner.FadeOut(fadeDuration);
 
                     break;
             }
         }
 
-        private (double startTime, double duration) getFadeOutParameters(DrawableOsuHitObject drawableObject)
+        private (double startTime, double fadeDuration) getFadeOutParameters(DrawableOsuHitObject drawableObject)
```

Comment:
If we're going for these sorts of renames then `applyState` above deconstructs to `(fadeStartTime, fadeDuration)`. The first name doesn't match.

Reply:
Commit [a4c1b9a](https://github.com/ppy/osu/pull/12811/commits/a4c1b9a1a7195dca0f4b5f5f078774f306c0c916)